docs/about-lightning.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Lightning (Coming soon)
+sidebar_label: Lightning
+---
+
+## The Lightning pitch
+
+##### Leveraging the tech powering the field-tested enterprise OpenFn platform...
+
+Lightning brings together the tried and tested technology we have been using
+since 2015 (OpenFn core, engine, adaptors and code from the OpenFn SaaS
+platform) to manage the orchestration and execution of integrations in a stable,
+scalable and secure way.
+
+##### ...but providing a fully open source web app with a user-friendly web interface.
+
+A fully open source web app, it can be deployed anywhere through Docker and
+comes with a user-friendly, low-code interface with the full functionality
+needed for organizations to build, run and audit their workflows all in one
+place. ![Lightning](/img/lightning_build_run_audit.png)
+
+##### Build
```

Comment:
If we have them, it would be great to add screesnhots for the `Build` and `Audit` interfaces to people can visual what's coming